### PR TITLE
BNF-994 Update html2json to handle second level topics

### PR DESCRIPTION
**Pull request by imranazad**
https://nicedigital.atlassian.net/browse/BNF-994

### CODE REVIEW

File: tasks/html2jsonld.js [hunk 1]
```diff
@@ -39,9 +39,19 @@ module.exports = (grunt) => {
...
+    rootPath = rootPath.replace(".\\",'').replace(/\\/g,"/") + "/";
+    var segments = path.dirname(file).replace(rootPath, '').split("/");
+    var directory;
+    if(segments.length == 2) // level 2 topic
+    {
+      directory = segments[0] + "/" + segments[1]
+    } else {
+      directory = segments[0];
+    }
+
```
Comment (chunter) [inline tasks/html2jsonld.js]: Maybe:
var directory = segments[0];
if(segments.length === 2) // level 2 topic
 directory = directory + "/" + segments[1];

reduce number of lines of code?

File: tasks/html2jsonld.js [hunk 2]
```diff
@@ -39,9 +39,19 @@ module.exports = (grunt) => {
     }
   }
   
-  function setPath(url, file, anchor, selector)
+  function setPath(url, rootPath, file, anchor, selector)
   {
-    return url + path.dirname(file).split("/").slice(-1)[0] + "/" + 
+    rootPath = rootPath.replace(".\\",'').replace(/\\/g,"/") + "/";
+    var segments = path.dirname(file).replace(rootPath, '').split("/");
```
Comment (chunter) [inline tasks/html2jsonld.js]: Dunno if there are any tests for the .js but if so i'd like to see what happens if the parameters are null, undefined or empty as there doesn't seem to be any validation

**Review by wa-rren-dev — APPROVED** [review]

File: tasks/html2jsonld.js (hunk 1 above)
Comment (imranazad) [reply tasks/html2jsonld.js]: Good shout! Have made the suggestion.

File: tasks/html2jsonld.js (hunk 2 above)
Comment (imranazad) [reply tasks/html2jsonld.js]: Another good shout, have added validation. No test I'm afraid. ;-(

**Review by chunter — APPROVED** [review]

**Pull request merged**